Makefile

```
@@ -80,6 +81,15 @@ build:
 	@[ "${GIT_HASH}" ] || ( echo ">> GIT_HASH is not set"; exit 1 )
 	go build -ldflags "-X main.GitTag=$(GIT_TAG) -X main.GitHash=$(GIT_HASH) -s -w" ./cmd/telliot
 
+.PHONY: generate-config-docs
+generate-config-docs:
```

Lets add a help comment here like for the other commands to explain what this script does exactly.
```suggestion
generate-config-docs: ## Use the example env file and the actual golang config struct to generate the setup doc page. .... <- add more info how it works.
```